scipy/optimize/_constraints.py

```diff
@@ -249,6 +250,57 @@ def __init__(self, constraint, x0, sparse_jacobian=None,
         self.bounds = (lb, ub)
         self.keep_feasible = keep_feasible
 
+    def is_feasible(self, x):
+        """
```

Comment:
The formatting is sort of unconventional. I'm not sure if it very important, but I would stick to PEP8 convention starting with one-line summary.

Something like, "Check if the constraint is satisfied.". I guess that's enough.